packages/create-workleap/src/generateProject.ts

```diff
@@ -0,0 +1,55 @@
+import child_process from "node:child_process";
+import process from "node:process";
+import type { TemplateId } from "./templates.js";
+
+interface UserInputs {
+    templateId: TemplateId;
+    outputDir: string;
+    packageScope?: string;
+    hostScope?: string;
+}
+
+export async function generateProject({ templateId, outputDir, packageScope, hostScope }: UserInputs) {
+    let commandName;
+    const args: string[] = [];
+
+    if (outputDir) {
+        args.push("-o", outputDir);
+    }
+
+    switch (templateId) {
+        case "host-application":
+            commandName = "host-application";
+            if (packageScope) {
```

Comment:
Aren't our ESLint rules enforcing a space between `commandName` and the `if` ?

Also, `packageScope` and `hostScope` are always required.

Reply:
> Aren't our ESLint rules enforcing a space between commandName and the if ?

Nope